package.json

```diff
@@ -19,6 +19,7 @@
     "@mantine/hooks": "^7.7.1",
     "firebase": "^10.10.0",
     "mantine-form-zod-resolver": "^1.1.0",
+    "firebase": "^10.10.0",
```

Comment:
Duplicate key here. Think one of these `firebase` properties can be removed. I'll push a quick commit to fix 😄